src/clojure/yaw/reaction.clj

```diff
@@ -50,3 +50,61 @@
                                        ;; (println "Update! " delta-time " ms")
                                        (swap! ratom (fn [_] delta-time)))))
     ratom))
+
```

Comment:
Some comments should be inserted, to explain things a little bit... also docstrings for functions